setup/azurecmdfiles.wxi

```diff
@@ -462,9 +462,6 @@
                         <Component Id="cmp6BDCB170E1F314882CF58F8F80DE76A6" Guid="*">
                             <File Id="filBFDE1699047559AB8DA89E0F660D35DF" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Billing\Microsoft.Azure.Commands.Common.Authentication.dll" />
                         </Component>
-                        <Component Id="cmpED253190CB7DAF0A995567435152DC6A" Guid="*">
```

Comment:
These should not be deleted.  We need the xml help files in the installer, or there will be no online help